app/Resources/views/base.html.twig

```diff
@@ -398,14 +407,17 @@
                             <a href="{{ path('app_facebook_index') }}">Filtre photo Facebook</a>
                         </li>
                     </ul>
+                    {% endif %}
                     <ul class="list__links list__links--col list__links--no-decor">
                         <li><h4><a href="https://twitter.com/hashtag/TousEnMarche" class="text--bold text--medium-small" target="_blank" rel="noopener noreferrer">#TousEnMarche</a></h4></li>
                         <li><a href="https://boutique.en-marche.fr" target="_blank" rel="noopener noreferrer">La Boutique</a></li>
                         <li><a href="http://vision-macron.fr" target="_blank" rel="noopener noreferrer">Vision Macron</a></li>
                         <li><a href="http://www.lesjeunesavecmacron.fr" target="_blank" rel="noopener noreferrer">Les Jeunes avec Macron</a></li>
                         <li><a href="http://www.leselusavecmacron.fr" target="_blank" rel="noopener noreferrer">Les élus avec Macron</a></li>
                         <li><a href="http://enmarcheletour.fr" target="_blank" rel="noopener noreferrer">En Marche Le Tour</a></li>
+                        {% if not app.request.attributes.get('_campaign_expired') %}
```

Comment:
You should have set a variable at the beginning of the template to avoid repeating those calls.